pkg/3scale/amp/template/amp.go

```diff
@@ -29,7 +29,7 @@ func (f *AmpTemplateFactory) Adapters() []adapters.Adapter {
 		adapters.NewBackendAdapter(),
 		adapters.NewMysqlAdapter(),
 		adapters.NewMemcachedAdapter(),
-		adapters.NewSystemAdapter(),
+		adapters.NewSystemAdapter(component.SystemFileStorageTypePVC),
```

Comment:
As you can see, we now create the System adapter with the desired fileStorage type depending on the template